<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestUI</class>
 <widget class="QDialog" name="TestUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тестирование</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ui_icon.qrc">
    <normaloff>:/calculator.png</normaloff>
    <normalon>:/calculator.png</normalon>
    <activeon>:/calculator.png</activeon>:/calculator.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>82</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Начало</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>111</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>16</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Перед началом тестирования введите пожалуйста своё имя</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>49</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>86</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>86</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>вопрос</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>171</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Вопрос</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Правильный ответ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ваш ответ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">+\-</string>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ui_icon.qrc"/>
 </resources>
 <connections/>
</ui>
